crs_application/src/task_managers/scan_acquisition_manager.cpp

```diff
@@ -53,14 +58,33 @@ ScanAcquisitionManager::~ScanAcquisitionManager()
 
 common::ActionResult ScanAcquisitionManager::init()
 {
-  RCLCPP_WARN(node_->get_logger(),"%s not implemented yet",__PRETTY_FUNCTION__);
+  // parameters
+  //scan_positions_ = node_->declare_parameter("scan_positions", std::vector<geometry_msgs::msg::Transform>());
+  framos_frame_id_ = node_->declare_parameter("framos_frame_id", "eoat_link");
+  max_time_since_last_point_cloud_ = node_->declare_parameter("max_time_since_last_point_cloud", 0.1);
+
+  // subscribers
+  point_cloud_sub_ = node_->create_subscription<sensor_msgs::msg::PointCloud2>("/point_cloud", 1,
+    std::bind(&ScanAcquisitionManager::handlePointCloud, this, std::placeholders::_1));
+
+  // service client
+  // todo(ayoungs): wait on service?
+  call_freespace_motion_client_ = node_->create_client<crs_msgs::srv::CallFreespaceMotion>("test_plan");
```

Comment:
ideally we would wait on the service to become available and return false if it times out, for now I'd be ok to print a warning at least if it facilitates testing

Reply:
okay, I'll update accordingly